### DISCUSSION TITLE
Email to GitHub support is getting bounced do to some organizational settings.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#123485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Well i tried contacting GitHub regarding my GitHub account (@krishealty) that was banned without any notice. It had all of my important repositories, but the support was ignoring me (Ticket ID: 2779325) so I tried the support email and below is the error im getting.

Can somebody help???

![Screenshot_2024-05-12-11-40-19-13_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/community/community/assets/169514951/6d3b8d7b-ad6b-4633-a763-2f01051e9471)

**Comment by CiceroLino** [comment]
Have you any thing illegal in your repository? Like malware? But, one thing is certain, dont rely too much in cloud. Even I, have almost all the my things in a external HD.

In cloud, just some copies

**Reply by ghost** [reply]
hi thanks for replying 

not really a virus, just some practice codes. Yes i shouldn't rely too much on cloud, but the support being ignoring me is disappointing. I get the "too many requests every week problem" but still.

**Reply by CiceroLino** [reply]
Strange, github deleting accounts... this not seems thing what github would make. Have you sure that you its not been hacked?

**Reply by rshgithub** [reply]
Well, I had the same issue. I got a virus on my laptop and lost access to my GitHub email for a while. Then, I suddenly discovered an unknown repository on my GitHub account, which I promptly deleted. However, I later found out that my GitHub account was flagged and reported as spam. I've been unable to restore my account ever since. While I can sign in, I remain invisible to others, and none of my repositories are visible either.

I've tried reaching out to the support team through various emails, but I keep receiving the same response, and I haven't been able to reach them so far.

I really need my account back as soon as possible, but it seems impossible to get in touch with anyone for help

**Reply by rshgithub** [reply]
it's not deleting accounts , the account is just reported as a flag / spam , the account and the repositories remains invisible for others but the owner still can sign in .

**Reply by ghost** [reply]
i dont think it would show the below error if it was hacked.


![Screenshot_2024-05-12-13-02-47-53_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/169514951/08ab8162-f216-464e-9020-e5d0e9c9d99d)

**Reply by rshgithub** [reply]
how could we contact the support team if our messages and emails are blocked ?

**Reply by rshgithub** [reply]
Anyone can help with this issue ???!

**Comment by rshgithub** [comment]
same issue here while trying to unflag my account by contacting the support team .
i still have the issue for more than two weeks and have no idea how to contact them !!

**Comment by LiteBrite82** [comment]
Hi @krishverse 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by rshgithub** [reply]
Hi @LiteBrite82  , i hope you are doing well
I would like to know if there's any available way to contact support team as we can't reach them by email nor they're not responding to our tickets which was posted a long time ago .
Please help if you can , i need my account back as soon as i can .
